app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -56,6 +56,35 @@ def api_sync
         end
         redirect_back(fallback_location: spree.admin_taxjar_settings_path)
       end
+
+      def cached_tax_categories
+        Rails.cache.fetch(
+          :tax_categories,
+          expires_in: 30.days
```

Comment:
Should we use the `SuperGood::SolidusTaxjar.cache_duration` here as well?

Reply:
The default cache duration is 2 hours, but in this case we don't want to update the tax categories this frequently. Users can manually refresh the tax categories.